pkg/actors/actors_test.go

```diff
@@ -700,6 +702,41 @@ func TestDeleteState(t *testing.T) {
 	assert.Nil(t, response.Data)
 }
 
+func TestCallLocalActor(t *testing.T) {
+	const (
+		testActorType = "pet"
+		testActorID   = "dog"
+		testMethod    = "bite"
+	)
+
+	req := invokev1.NewInvokeMethodRequest(testMethod).WithActor(testActorType, testActorID)
+
+	t.Run("invoke actor successfully", func(t *testing.T) {
+		testActorRuntime := newTestActorsRuntime()
+		resp, err := testActorRuntime.callLocalActor(context.Background(), req)
+		assert.NoError(t, err)
+		assert.NotNil(t, resp)
+	})
+
+	t.Run("actor is already disposed", func(t *testing.T) {
+		testActorRuntime := newTestActorsRuntime()
+		actorKey := testActorRuntime.constructCompositeKey(testActorType, testActorID)
+		act := newActor(testActorType, testActorID)
+		testActorRuntime.actorsTable.LoadOrStore(actorKey, act)
+		act.lock()
+		assert.True(t, act.isBusy())
+		ch := act.channel()
+		act.unlock()
+		_, closed := <-ch
```

Comment:
rename closed to more. Preferably add comment on what the second return value is.

Reply:
I would not add comment for golang channel syntax... closed returns channel status.